tweak: include note for allow db/schema as .* (#7656)
fishtown-analytics/docs.getdbt.com: website/docs/docs/explore/external-metadata-ingestion.md

    - *Optional*: Enable additional features such as **cost optimization** in the **Features** section under **Platform metadata credentials**.
6. Under **Catalog filters**, apply filters to restrict which metadata is ingested:
    - You can filter by **database**, **schema**, **table**, or **view**.

>       - Note: To include all databases or schemas, use `.*` as the allow value.

Reviewer: ```suggestion
      - **Note:** To include all databases or schemas, enter `.*`  in the **Allow** field.
```
Hi @jcserv , please review if this suggested change is accurate. Thanks!
